src/backend/web/app.js

```diff
@@ -19,6 +24,15 @@ app.use(logger);
 
 app.use(cors());
 
+const server = new ApolloServer({
+  typeDefs,
+  resolvers,
+  context: { storage },
+});
+
+// Add the Apollo server to app and define GraphQL's endpoint
+server.applyMiddleware({ app, path: '/gql' });
```

Comment:
Can we change this to `/graphql` to match the more common/expected name APIs seem to use?

From [the docs]:

> a GraphQL server operates on a single URL/endpoint, usually /graphql

For example, [GitHub uses https://api.github.com/graphql](https://developer.github.com/v4/guides/forming-calls/#the-graphql-endpoint).

Reply:
Changed to `/graphql`